### PR TITLE
Realtime output in runLocally function

### PR DESCRIPTION
**Pull request by yannschepens**
…play debug output with this mode

### CODE REVIEW

File: src/functions.php [hunk 1]
```diff
@@ -314,7 +314,15 @@ function runLocally($command, $timeout = 60)
 
     $process = new Symfony\Component\Process\Process($command);
     $process->setTimeout($timeout);
-    $process->run();
+    $process->run(function($type, $buffer){
+        if (isVeryVerbose()) {
```
Comment (elfet) [inline src/functions.php]: May be it's better use `isDebug()` here?

File: src/functions.php [hunk 2]
```diff
@@ -322,7 +330,7 @@ function runLocally($command, $timeout = 60)
 
     $output = $process->getOutput();
 
-    if (isDebug() && !empty($output)) {
+    if (!isVeryVerbose() && isDebug() && !empty($output)) {
```
Comment (elfet) [inline src/functions.php]: And delete this duplicate output?

File: src/functions.php (hunk 1 above)
Comment (yannschepens) [inline src/functions.php]: As you wish, the twice are ok for me. I do the modification.

File: src/functions.php (hunk 2 above)
Comment (yannschepens) [inline src/functions.php]: Yes

**Comment by yannschepens** [comment]
Updated

File: src/functions.php [hunk 3]
```diff
@@ -322,7 +330,7 @@ function runLocally($command, $timeout = 60)
 
     $output = $process->getOutput();
 
-    if (isDebug() && !empty($output)) {
+    if (!isDebug() && !empty($output)) {
```
Comment (elfet) [inline src/functions.php]: Now this will be printing output all the time. Lets delete this block completely.

File: src/functions.php [hunk 4]
```diff
@@ -314,7 +314,15 @@ function runLocally($command, $timeout = 60)
 
     $process = new Symfony\Component\Process\Process($command);
     $process->setTimeout($timeout);
-    $process->run();
+    $process->run(function($type, $buffer){
+        if (isDebug()) {
+            if ('err' === $type) {
+                write('<fg=red>'.$buffer.'</fg=red>');
```
Comment (elfet) [inline src/functions.php]: Maybe it's better to keep this format?
```php
"<fg=red>></fg=red> $line"
```

File: src/functions.php (hunk 3 above)
Comment (yannschepens) [inline src/functions.php]: Ok

File: src/functions.php (hunk 4 above)
Comment (yannschepens) [inline src/functions.php]: You're the master, I change it. I do the same for the next line.

**Pull request merged**